src/components/Personalization/index.js

```diff
@@ -10,7 +10,8 @@ OF ANY KIND, either express or implied. See the License for the specific languag
 governing permissions and limitations under the License.
 */
 
-import { isNonEmptyArray } from "../../utils";
+import CONFIG_DOC_URI from "../../constants/docUri";
```

Comment:
It looks like this will link to the Tracking Events documentation page. Would you want to link to some sort of Retrieving Personalization Decisions page or something like that?

Reply:
yes, we will need a special URL for getDecisions command. Do you know how can we create a. adobe.ly URL?

Reply:
I think you can just go to https://bitly.com/ and paste in the documentation URL and it will spit out an adobe.ly URL. @liljenback, if the documentation structure shifts around later on, do you know if we can change the target URL that the adobe.ly URL redirects to?

Reply:
@Aaronius , it doesn't look like you can update auto-generated links. In any case if documentation links mostly stay within alloy it seems like from a maintenance aspect it would be easier just generating new links and updating alloy. If we feel like these links need to work as well in old libraries we will have to switch to creating named links (e.g. we pick the name adobe.ly/alloy-events) which would enable us to update them if the structure changes.

Reply:
I vote that we'll just create new links if our doc structure change.

Reply:
What would be the downsides with using named links? How do we create and update them? It seems like being able to update them would be the optimal way to go, all other things being equal.